samples/layouts/dock-manager/styling/src/index.ts

```diff
@@ -125,16 +160,19 @@ export class DockManagerStylePanes {
 
         this.dockManager.addEventListener("closePane", () => this.handleClosePane());
         this.dockManager.addEventListener("pinPane", () => this.handlePinPane());
+
+        registerIcon("arrow-down", "https://fonts.gstatic.com/s/i/materialiconsoutlined/arrow_drop_down/v15/24px.svg", "material");
+        registerIcon("arrow-up", "https://fonts.gstatic.com/s/i/materialiconsoutlined/arrow_drop_up/v12/24px.svg", "material");
     }
 
     handleClosePane() {
         this.dockManager.activePane.hidden = true;
-        this.dockManager.layout = {...this.layout};
+        this.dockManager.layout = { ...this.layout };
```

Comment:
We should use the actual reference of the layout. Same for `handlePinPane`.
```suggestion
        this.dockManager.layout = { ...this.dockManager.layout };
```